recipe/windows-bat.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/graphviz/backend.py b/graphviz/backend.py
+index 955551a..be35745 100644
+--- a/graphviz/backend.py
++++ b/graphviz/backend.py
+@@ -69,6 +69,8 @@ def command(engine, format, filepath=None):
+     if format not in FORMATS:
+         raise ValueError('unknown format: %r' % format)
+ 
++    if PLATFORM == 'windows':
++        engine = '%s.bat' % engine
```

Comment:
If you [re-render]( https://github.com/conda-forge/conda-smithy#re-rendering-an-existing-feedstock ) with `conda-smithy` 2.2.2, the AppVeyor issue should go away.